src/web-push-lib.js

```diff
@@ -201,6 +223,10 @@ WebPushLib.prototype.generateRequestDetails =
         requestDetails.headers.Authorization = 'key=' + currentGCMAPIKey;
       }
     } else if (currentVapidDetails) {
+      if (contentEncoding === WebPushLib.supportedContentEncodings.AES_128_GCM
+          && subscription.endpoint.indexOf('https://fcm.googleapis.com') === 0) {
+        subscription.endpoint = subscription.endpoint.replace('fcm/send', 'wp');
```

Comment:
@gauntface is this reliable?

Reply:
For now it is. We need to do this until FCM actually supports it on the normal URL.